debian/control

```diff
@@ -226,7 +228,9 @@ Depends: ceph-base (= ${binary:Version}),
 Recommends: ceph-mgr-dashboard,
             ceph-mgr-diskprediction-local,
             ceph-mgr-k8sevents,
-            ceph-mgr-cephadm
+            ceph-mgr-cephadm,
+            nvme-cli,
+            smartmontools,
```

Comment:
i was searching for "smartctl" and "scrape" in the source tree to find where "smartmontools" is used. and i ended up finding `block_device_run_smartctl()`, where "block_device_run_vendor_nvme()" is also found.

could you expand in the commit message to explain the connection between nvme-cli, smartmontools and the scrape feature? for instance, where these tools are called, and what happens if they are missing.

Reply:
@tchaikov @jdurgin sure, I'll update both commit messages to include all explanations. I separated the commits on purpose (in case we want to revert only debian changes, for example). Does it make sense to have a single commit here?